common/surroundings_loader.rs

```diff
@@ -59,11 +26,11 @@ pub enum LoadType {
 /// Iteratively load points cube-shaped layers around the some point.
 /// That point can be updated with calls to `update`.
 /// What "load" exactly means depends on the closures provided.
-pub struct T {
+pub struct SurroundingsLoader {
```

Comment:
I've adopted the ocamly convention of using `modname::T` instead of `modname::Modname` - it means doing less importing at the top of every module that uses the type.

Reply:
Will rename it back then.

Reply:
Cool thanks